### DISCUSSION TITLE
CMake build with Eigen

### DISCUSSION THREAD

**Discussion by yuankipovich** [BlueBrain/HighFive#699] [Questions]
When I try to compile HighFive with Eigen using Cmake, this error appears:
>-- Found Eigen 3.4.90: 
CMake Error at CMake/HighFiveTargetDeps.cmake:53 (message):
  Eigen was requested but could not be found
Call Stack (most recent call first):
  CMakeLists.txt:63 (include)

What should I do to build a library with Eigen?

**Comment by 1uc** [comment]
Unfortunately, this is way too little information to help you. We'll at the very least need to know which version of HighFive and Eigen you're using (is 3.4.90 really the one you're trying to use?); where is Eigen installed; what operating system are we dealing with; how are you calling `cmake`; did you delete the build directory; etc.

We build HighFive with Eigen during CI, which should ensure that it works on Ubuntu. You can look at those instructions:
https://github.com/BlueBrain/HighFive/blob/master/.github/workflows/ci.yml#L52
https://github.com/BlueBrain/HighFive/blob/master/.github/build.sh#L11

If you're using Eigen and HighFive you should be aware of the following https://github.com/BlueBrain/HighFive/issues/532
